src/app/layouts/admin-menu/admin-menu.component.html

```diff
@@ -0,0 +1,17 @@
+<mat-nav-list class="admin-menu" role="navigation" aria-label="navigation">
+    <a mat-list-item 
+       *ngFor="let item of adminMenu; let i = index" 
+       (click)="menuItemSelected(item.id)"
+       (keyup.enter)="menuItemSelected(item.id)" 
+       class="admin-menu__list-item"
+       [ngClass]="{'admin-menu__list-item--selected': item.isSelected}" 
+       role="menuitem"
+       [attr.aria-selected]="item.isSelected ? 'true' : 'false'" 
+       [attr.tabindex]="i + 1">
```

Comment:
Is tabindex required ? This is an anchor tag for which focus works by default

Reply:
not required.removed from the code.